learnable/src/components/LoginForm/LoginForm.jsx

```diff
@@ -20,11 +20,10 @@ function LoginForm() {
 
     const handleSubmit = (event) => {
         event.preventDefault();
-
-        if (credentials.email && credentials.password) {
+        if (credentials.username && credentials.password) {
             postData().then((response) => {
             window.localStorage.setItem("token", response.token);
-            navigate("/explore");
+            navigate("/");
             });
```

Comment:
Awesome, thanks! I had done this in my first PR but it was reverted with some merge conflict issues and I missed re-adding this piece yesterday afternoon. Good catch, thanks